Sources/APIConnect/Model.swift

```diff
@@ -18,6 +18,10 @@ public protocol APIConnectEnvironment {
 }
 
 extension APIConnectEnvironment {
+    public static func get(key: String) -> String? {
```

Comment:
I prefer to use
`Environment.isDebugMode`
Which a computed property for
`Environment.get(key: "debugMode") == "true"`
Not to hardcode these kind of think in many places.